packages/connect-popup/webpack/dev.webpack.config.ts

```diff
@@ -1,7 +1,13 @@
 import path from 'path';
+import { DefinePlugin } from 'webpack';
+import { execSync } from 'child_process';
+
 import { merge } from 'webpack-merge';
 import { WebpackPluginServe } from 'webpack-plugin-serve';
 import prod from './prod.webpack.config';
```

Comment:
`dev.webpack.config.ts` extends `prod` config. So I believe that changes to this file could be removed

Reply:
resolved